### ISSUE TITLE
Profile pictures.

### ISSUE THREAD

**Issue by wizardsambolton** [GravityProject/gravity#84]

**Comment by pmuens** [comment]
Do you have an idea how to implement that? Store the pictures on the server or use something such as Filepicker?

**Comment by onedaylove** [comment]
@pmuens Do you think something like Gravatar could be used? What about using Facebook or Google login and have it just ask for the user's profile picture?

**Comment by zaverichintan** [comment]
We can use amazon or on own server. By file picker in front end and Backend with meteor file collection

**Comment by pmuens** [comment]
Yes. file picker sounds good. Gravity is hosted on Heroku so I can add it to the application.

**Comment by zaverichintan** [comment]
Meteor can use - https://atmospherejs.com/mrt/impicker-security .
Kindly suggest.

**Comment by pmuens** [comment]
Looks great. We've used this before https://atmospherejs.com/goltfisch/filepicker

Maybe we wait until Meteor 1.3 so that we can use the official NPM package...

**Comment by zaverichintan** [comment]
I think this is solved ? Or I am only getting the profile picture ?

**Comment by pmuens** [comment]
We've currently only used Gravatar. So you profile picture is taken from there.
